tests/Element/TagInlineTest.php

```
@@ -497,319 +488,308 @@ public function testRenderWithLoadDefault(): void
     }
 
     public function testRenderWithUserOverridesGlobalDefaults(): void
     {
-        $previous = SimpleFactory::getDefaults(TagInline::class);
+        SimpleFactory::setDefaults(
+            TagInline::class,
+            [
+                'class' => 'from-global',
+                'id' => 'taginline-id-global',
+            ],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithUserOverridesGlobalDefaults'. (undefined)

(StaticAccess)